<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDictionaries</class>
 <widget class="QDialog" name="DownloadDictionaries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Dictionaries</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="availableListLabel">
     <property name="text">
      <string>Available Dictionary Downloads</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dictTableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadListButton">
       <property name="text">
        <string>Update Dictionary List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download Dictionaries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
